mkosi

```diff
@@ -2711,6 +2713,9 @@ def process_setting(args: CommandLineArguments, section: str, key: str, value: A
         elif key == "Bootable":
             if args.bootable is None:
                 args.bootable = parse_boolean(value)
+        elif key == "BootProtocols":
+            list_value = value if type(value) == list else value.split()
```

Comment:
The command line option should override, not extend this.

Reply:
done